### PR TITLE
[feature] introduce fluentd_flags and fluentd_plugin_dir

### LINKED ISSUE #34: the role does not manage fluentd flags
some configuration options are only present in command line flags, not in `fluentd.conf`, such as `-p`.

the role should manage flags.

### PR DESCRIPTION
**Pull request by trombik**
fixes #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +90,7 @@ None
   roles:
     - ansible-role-fluentd
   vars:
+    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% endif %}"
```
Comment (mitsururike) [inline README.md:93]: if -p option is mandatory for FreeBSD, __fluentd_flags should be defined in vars/FreeBSD.yml.

Comment (trombik) [reply README.md:93]: `-p` is not necessarily needed. it is only when you want to install _local_ plug-ins *AND* `fluentd` behaves same on all distributions. the port should have patched the gem to include `/usr/local/etc/fluentd/plugin`.

Comment (mitsururike) [reply README.md:93]: i think it is better to add -p option when fluentd_plugins_to_install is not empty, or it should be documented that users need to set -p option when fluentd_plugins_to_install is not empty.

**Review by mitsururike — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (trombik) [reply README.md:93]: there are two way to manage plug-ins: one is using `fluent-gem` and another is placing plug-in files under `plugin` directory.  `fluent-gem` does not install plug-ins to `plugin` directory. `plugin` directory in question is for those plug-ins that are locally-brewed. the former requires you to create a gem, on the other hand, the latter does not. See:
http://docs.fluentd.org/v0.12/articles/plugin-management#add-a-plugin-via-etcfluentplugin

Comment (trombik) [reply README.md:93]: https://github.com/reallyenglish/ansible-role-fluentd/pull/36 documents when `fluentd_flags` is necessary.

**Pull request merged**